assets/scss/01-atoms/global/_elements.scss

```diff
@@ -17,9 +17,12 @@
 }
 
 html {
-  @media ($bp-small-max){
-    font-size: 14.5px;
+  font-size: 14.5px;

```

Comment:
ask @clairesunstudio  re: font-size in px vs. rem

Reply:
For fonts I think we want to standardize `rem`

Reply:
these font sizes are in pixels because they are being used to set the rem values for the rest of the site